cmd/influxd/run/config_test.go

```diff
@@ -141,6 +141,8 @@ bind-address = ":2010"
 [[udp]]
 bind-address = ":4444"
 
+[[udp]]
```

Comment:
Allowing configuration of multiple listeners through env var overrides would require more substantial changes to `applyEnvOverrides`.